Assets/Scripts/Fire.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
+
+public class Fire : MonoBehaviour {
+    public float scaleFire;
+    public float speedDreasing;
+    public float minScale;
+    public int densityFire;
+    private ParticleSystem part;
+    private ParticleSystem.ShapeModule sh;
+    private ParticleSystem.EmissionModule em;
+
+    void Start() {
+        part = this.GetComponent<ParticleSystem>();
+        sh = part.shape;
+        em = part.emission;
+        sh.scale = Vector3.ClampMagnitude(Vector3.one, scaleFire);
+        em.rateOverTime = (ParticleSystem.MinMaxCurve)(System.Math.Pow(scaleFire, 3) * densityFire);
+    }
+
+    void Update() {
+        Collider[] woods = Physics.OverlapSphere(this.transform.position, sh.scale.x + 1);
+        foreach (var wood in woods) {
+            if (!part.isStopped && wood.gameObject.GetComponent<Wood>()) {
+                sh.scale += Vector3.ClampMagnitude(Vector3.one, wood.gameObject.GetComponent<Wood>().size);
+                em.rateOverTime = (ParticleSystem.MinMaxCurve)(System.Math.Pow(sh.scale.magnitude, 3));
```

Comment:
You can remove pair of brackets

Reply:
It is to change the type of  System.math... from int to MinMaxCurve so that is not a method and I havwe to keep the parentheses